src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -62,7 +73,30 @@ class EntityClient (requestContext: RequestContext) extends Actor {
         requestContext.failWith(
           new RequestProcessingException(StatusCodes.InternalServerError, error.getMessage))
     }
+  }
 
+  def createEntities(wsNamespace: String, wsName: String, entities: Seq[Entity]) = {
+    val results = entities.map(entity => createEntityOrReportError(wsNamespace, wsName, entity))
+    // have to manually construct an HttpResponse here because we're aggregating the results of multiple calls to Rawls
+    val response = HttpResponse(OK, HttpClient.createJsonHttpEntity(results.toJson.compactPrint))
```

Comment:
Not sure what the right thing to do here is - should we return ```OK``` for the batch operation even if there are some individual failures, or only if everything succeeds? And does ```OK``` or ```Created``` make more sense in that case?